README.md

```diff
@@ -97,6 +97,13 @@ const { data: decryptedData, verified } = await decryptMessage({
 **For streamed inputs:**
 to encrypt (and/or sign), pass the stream to `textData` or `binaryData` based on the streamed data type. Similarly, to decrypt and verify, the input options are the same as the non-streaming case. However, if `armoredMessage` (or `binaryMessage`) is a stream, the decryption result needs to be handled differently:
 ```js
+// explicitly loading stream polyfills for legacy browsers is required since v7.2.2
+if (!globalThis.TransformStream || !globalThis.WritableStream) {
+  const ponyfilledStreams = await import('web-streams-polyfill/es6');
```

Comment:
The polyfill already overwrites the global variables. If you want to do so manually (which I think is reasonable for clarity), you need the ponyfill.
```suggestion
  const ponyfilledStreams = await import('web-streams-polyfill/ponyfill/es6');
```